apps/notifications/src/standalone/index.jsx

```diff
@@ -12,6 +13,24 @@ const localePattern = /[&?]locale=([\w_-]+)/;
 const match = localePattern.exec( document.location.search );
 const locale = match ? match[ 1 ] : 'en';
 
+const fetchLocale = ( localeSlug ) => {
+	const xhr = new XMLHttpRequest();
+
+	xhr.open( 'GET', `https://widgets.wp.com/languages/notifications/${ localeSlug }.json`, true );
```

Comment:
Nowadays you can do this with `fetch()`, it's much more elegant, promisified etc.

Reply:
Updated [here](https://github.com/Automattic/wp-calypso/pull/93390/commits/5195d99e3d62ffa9b55a6e24d91d1a63d17e18a8)